src/util/get-config-files.ts

```diff
@@ -45,26 +47,27 @@ export async function getConfigFiles(
       path,
     });
 
-    return [requestedRepoFile, defaultRepoConfig];
+    files.push(defaultRepoConfig);
   }
 
+  const file = files[files.length - 1];
+
   // if the configuration has no `_extends` key, we are done here.
-  if (!requestedRepoFile.config._extends) {
-    return [requestedRepoFile];
+  if (!file.config?._extends) {
```

Comment:
this will break in Node 10 which the current version still supports

Reply:
Replaced with `if (!file.config || !file.config._extends) {`, let's see now.

Reply:
@gr2m This should be really it now... (I know I said that before) Approval needed....